### ISSUE TITLE
[Accessibility]Inspect: In View is Details, the yellow rectangle jumps to the front item automatically when we focus on the sub items in form individually 

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#4816]
* .NET Core Version: 
.Net 6.0.100-preview.4.21218.5 from main branch

* Have you experienced this same bug with .NET Framework?: 
Yes

**More info:**
1.	This issue can also repro on .Net Core 3.1, 5.0 and .NET framework 4.7.1.
2.	This issue cannot happen every time.
3.	This issue cannot happen if we focus sub item in inspect.
4.	This issue can repro no matter enable virtual mode or disable virtual mode.

**Minimal repro:**
1.	Extract the “noVirtual_Listview_detail.zip” and open it in VS.
[noVirtual_Listview_detail.zip](https://github.com/dotnet/winforms/files/6340138/noVirtual_Listview_detail.zip)
2.	Build and run this application.
3.	Start Inspect and focus on one ListView item(e.g. item1) by clicking left mouse, then focus on one ListView subitem(e.g. sub1) by clicking left mouse.
4.	Observe the yellow rectangle.

**Actual behavior:**
The yellow rectangle jumps to the front item automatically when we focus on the sub items in form individually.
![issue_repro](https://user-images.githubusercontent.com/45864017/115327220-236c6b00-a1c1-11eb-867d-c3e06c2e426a.gif)

**Expected behavior:**
The yellow rectangle cannot jump to the front item automatically when we focus on the sub items in form individually, it should focus on subitem.

**Comment by Lydia-Shi** [comment]
Verified this issue with .NET 7.0.100-alpha.1.22060.5, this issue is fixed. The yellow rectangle still focus on subitem when setting View to Details for ListView control and focus on sub items.
![SubItems](https://user-images.githubusercontent.com/54019172/148904780-11b4713b-9cf2-4993-887c-3b33a5dac30f.gif)
